manifests/g/GnuWin32/Tar/1.13-1/GnuWin32.Tar.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: GnuWin32.Tar
+PackageVersion: 1.13-1

```

```suggestion
PackageVersion: 1.13
```
This must match Control Panel.